### ISSUE TITLE
Is there an open source plan for pretrained discriminator?

### ISSUE THREAD

**Issue by liyaowei-stu** [Luo-Yihong/YOSO#9]
Awesome work!

Do you have any open source plan for pretrained discriminator?

**Comment by Luo-Yihong** [comment]
Currently, we do not have plans to release the pre-trained discriminator. I would like to say that initializing the discriminator from half-Unet is effective enough.

**Comment by liyaowei-stu** [comment]
ok, thanks! 

I found lambda_con=2, ambda_kl=0.5 in the default setting. Is it necessary?
accelerator.backward(args.lambda_con * loss + args.lambda_kl * kl_loss + errG_gan)

**Comment by Luo-Yihong** [comment]
You may adjust the hyper-parameters. Empirically speaking, larger lambda_kl results in better multi-step performance and worse one-step performance.
